code/modules/projectiles/guns/magic/staff.dm

```diff
@@ -54,3 +54,108 @@
+	if(revenant_level == 3)
+		if(revenant_souls >= 20)
+			user << "<font color=purple>You only give a quick glimpse at the staff, as you hear the screams of the fallen emenating from the crystal mounted ontop of the staff, echoing throughout the station. Your powers grows even stronger...</font>"
+			max_charges = 5
+			charges = 5
+			recharge_rate = 6
+			chambered = new /obj/item/ammo_casing/magic/staffofrevenant/level4(src)
+			revenant_level = 4
+			revenant_damage = 40
+			world << 'sound/hallucinations/i_see_you1.ogg'
+			world << "<font color=purple><b>\"Your end is near...\"</font>"
+	if(revenant_level == 4)
+		if(revenant_souls >= 25) // if you reach this point, you pretty much won already
+			user << "<font color=purple>Just merely thinking of the power you have acquired is enough to trigger the staff's final evolution... It's destructive powers lets out an even louder wailing than last time, so loud that it echoes throughout the entire station, alerting those still standing that its futile to resist now...</font>"
+			max_charges = 6
+			charges = 6
+			recharge_rate = 5
+			chambered = new /obj/item/ammo_casing/magic/staffofrevenant/level5(src)
+			revenant_level = 5
+			revenant_damage = 60
+			world << "<font size=5><font color=purple><b>\"UNLIMITED... POWER!\"</font>"
+			world << 'sound/hallucinations/wail.ogg'
+	if(revenant_level == 5)
+		if(revenant_souls >= 50) // if you reaaally go the extra mile to cement your victory
+			user << "<font color=purple>The Staff... Somehow, you managed to do what no necrolord had ever managed, to awaken the staff further than this... It does not even seem to react, but you can feel it! The staff, it has become so much more potent! None can stand in your way!</font>"
+			chambered = new /obj/item/ammo_casing/magic/staffofrevenant/level666(src)
+			max_charges = 15
+			charges = 15
+			recharge_rate = 1
+			revenant_level = 666
+			revenant_damage = 200
+			world << "<font size=5><font color=purple><b>COWER BEFORE ME MORTALS!</font>"
+			world << 'sound/hallucinations/wail.ogg'
+
+	if(revenant_level <= 4)
+		user << "<font color=purple><b>Your [src] has [revenant_souls] souls contained within. Your power will grow every fifth soul...</font color></b>"
+		user << "<font color=purple>It has a maximum charge of [max_charges], with a recharge rate of [recharge_rate]. Each projectile deals [revenant_damage] damage.</font color>"
+	if(revenant_level == 5)
+		user << "<font color=purple><b>Your [src] has [revenant_souls] souls contained within. Your power can only grow if you absorb a total of 50 souls...</font color></b>"
+		user << "<font color=purple>It has a maximum charge of [max_charges], with a recharge rate of [recharge_rate]. Each projectile deals [revenant_damage] damage.</font color>"
+	if(revenant_level == 666)
+		user << "<font color=purple><b>Your [src] has [revenant_souls] souls contained within. Your power can not possibly grow any further...</font color></b>"
```

Comment:
Also you need to terminate the bold before the font like:

`<font><b>hello</b></font>`